### PR TITLE
Update readme fixes #225

### LINKED ISSUE #225: elementary tweak should not be used anymore
It's not recommended to use elementary tweaks as it is [no longer maintained](https://plus.google.com/u/0/+MichaelStarkweather/posts/Y8zixFDEWiv) and has been known to break systems.
IMHO elementary+ shouldn't mention it in it's installation guide... what do you think?

### PR DESCRIPTION
**Pull request by cybre**

### CODE REVIEW

**Comment by cybre** [comment]
So, are we happy with this? @quassy @mank319  @RafaelCz?

File: README.md [hunk 1]
```diff
@@ -16,23 +15,19 @@ This is why I hope for this repository to become a central place to collect addi
...
-
 You can add our PPA (Personal Package Archive) to your sources:
 ```
 sudo add-apt-repository ppa:cybre/elementaryplus
 sudo apt-get update
 sudo apt-get upgrade && sudo apt-get install elementaryplus
 ```
-or just download the directory or clone the repository and run `./install.sh` from the parent directory of this repository.
-Then select the theme in [elementary tweaks](https://code.launchpad.net/~versable/elementary-community/elementary-tweaks).
+or just download the directory or clone this repository:
```
Comment (RafaelCz) [inline README.md]: - Why does this sentence end with a colon?
- Linking "download" to `https://github.com/mank319/elementaryPlus/archive/master.zip` would make it easier for GitHub unexperienced users to find the correct download link.

Comment (cybre) [inline README.md]: My bad.

Comment (cybre) [inline README.md]: Fixed.

**Comment by RafaelCz** [comment]
:thumbsup:

**Comment by quassy** [comment]
:+1:

**Comment by cybre** [comment]
@quassy Yes. What do you think about [this](http://i.imgur.com/fSPlCAD.png), though? We can replace the Plank screenshot with this image. I'll make one for the indicators, as well.

**Comment by RafaelCz** [comment]
Why is there the default PDF reader icon in this screenshot?

**Comment by cybre** [comment]
For some reason, we have the icon in our icon theme. Weird.

**Comment by RafaelCz** [comment]
I've pushed a nice Bitcoin icon into a few minutes ago... take this one instead ;)

**Pull request merged**